Package.swift

```diff
@@ -2,7 +2,7 @@
 import PackageDescription
 
 let tag = "0.0.108"
-let checksum = "e69f12f9eb3335864ccd336d1285ec5e90826d68ca94594958aa61281af3ef84"
+let checksum = "01eb13d93f90ae228130ad17b2dffe8388611886125f8dd53ba2d31518209f92"
```

Comment:
I've by adding the package to a sample app locally and it works with the new checksum! `01eb13d93f90ae228130ad17b2dffe8388611886125f8dd53ba2d31518209f92`

Reply:
amazing, thanks! I think @arik-so is tying some last loose ends and we're going to try to support SPM with 109. 

this PR clarifies the order for release, but ultimately will not be merged since we're leapfrogging :)